### PR TITLE
cromwell 0.14

### PR DESCRIPTION
**Pull request by slnovak**
This PR adds [Cromwell](https://github.com/broadinstitute/cromwell) to Homebrew. It's a workflow execution engine based on the Workflow Description Language.

### CODE REVIEW

File: Library/Formula/cromwell.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      }
+    EOS
+
+    (testpath/"hello.json").write <<-EOS
+      {
+        "test.hello.name": "world"
+      }
+    EOS
+
+    Open3.popen3("#{bin}/cromwell run hello.wdl hello.json") do |stdin, stdout, _|
```
Comment (mikemcquaid) [inline Library/Formula/cromwell.rb]: You can use `shell_output` instead of `popen3` here.

**Comment by slnovak** [comment]
@mikemcquaid: thanks! Updated the PR.

**Comment by apjanke** [comment]
Since this is just a jar file and wrapper script, would it be appropriate to do `bottle :unneeded` for this?

And could you squash this PR to a single commit, to keep the commit history tidy?

**Comment by dunn** [comment]
Can we compile it, instead of just installing the `.jar`?

**Comment by slnovak** [comment]
@apjanke: Done and done.

@dunn: What advantage would this have since the Cromwell project releases .jar's? We would need to install additional dependencies (i.e., Scala, sbt, etc).

**Comment by dunn** [comment]
We try to always give users the option to build from source, if they want.  We also generate our own binaries ("bottles"), so dependencies marked as `:build` won't have to be installed by most users.

**Comment by slnovak** [comment]
@dunn -- thanks for the explanation! I updated the formula to include building from source.

**Comment by rakkesh** [comment]
@slnovak, for the stable release, you need to download the [source tarball](https://github.com/broadinstitute/cromwell/archive/0.15.tar.gz], and have build commands, not just fetch the `jar` file.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `cromwell 0.14 (new formula)`.

**Pull request closed without merging**